package.json

```
@@ -22,8 +22,20 @@
   },
   "dependencies": {
     "@ethersphere/bee-js": "^0.5.0",
```

The `0.5.0` is not compatible with latest nodejs.
```suggestion
    "@ethersphere/bee-js": "^0.5.1",
```